### ISSUE TITLE
discordbotlist.com: detection

### ISSUE THREAD

**Issue by aaronliu0130** [uBlockOrigin/uAssets#20507]
### Prerequisites

- [X] This is **NOT** a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/19976), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://discordbotlist.com/bots/dank-memer/upvote`

### Category

detection

### Description

There's a red block telling to disable ad blocker after you click "upvote". The button then turns into a disabled "upvote anyway" that does nothing when clicked.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/uBlockOrigin/uAssets/assets/11874211/b5131726-3521-4b79-b977-e8e2d550953a)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.53.0
Firefox: 115
filterset (summary):
 network: 147909
 cosmetic: 225501
 scriptlet: 30933
 html: 1345
listset (total-discarded, last-updated):
 added:
  adguard-social: 21997-858, now
  fanboy-cookiemonster: 48861-3897, now
  fanboy-social: 16184-9, now
  ublock-annoyances: 5725-528, now
  CHN-0: 25516-143, now
  RUS-0: 40161-569, 1h.7m
  adguard-cookies: 28006-67, now
  adguard-popup-overlays: 25059-1603, now
  adguard-mobile-app-banners: 4729-1069, now
  adguard-other-annoyances: 13752-314, now
  adguard-widgets: 2340-1, now
  easylist-annoyances: 4165-117, now
  [5 lists not shown]: [too many]
 default:
  user-filters: 2-0, never
  ublock-filters: 36013-428, now
  ublock-badware: 7434-5, now
  ublock-privacy: 654-0, now
  ublock-unbreak: 2179-0, now
  ublock-quick-fixes: 137-1, 1h.6m
  easylist: 75145-826, now
  easyprivacy: 32837-833, now
  urlhaus-1: 12395-0, now
  plowe-0: 3721-1, now
filterset (user): [array of 2 redacted]
trustedset:
 added: [array of 4 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 655 ms (selfie)
 maxAssetCacheWait: 427 ms
popupPanel:
 blocked: 11
 network:
  bkrtx.com: 2
  btloader.com: 1
  cloudflareinsights.com: 2
  doubleclick.net: 1
  googlesyndication.com: 1
  googletagmanager.com: 1
  network-n.com: 2
  script.ac: 1
 extended:
  ##.ad
  ##+js(adjust-setInterval, countdown, *, 0.02)
```
</details>

**Comment by mapx-** [comment]
maybe you click "upvote" in this page ?
`https://discordbotlist.com/bots/dank-memer`

I see no warning / detection

**Comment by aaronliu0130** [comment]
It detects after you click upvote. And yes, account is required.

**Comment by mapx-** [comment]
> maybe you click "upvote" in this page ?
> https://discordbotlist.com/bots/dank-memer

**Comment by mapx-** [comment]
no detection

![image](https://github.com/uBlockOrigin/uAssets/assets/10303732/09736d0a-d419-4d29-91e2-58a4a059051c)

**Comment by aaronliu0130** [comment]
Are you sure you've pressed the upvote button?

**Comment by aaronliu0130** [comment]
> maybe you click "upvote" in this page ?
> `https://discordbotlist.com/bots/dank-memer`

That leads to the link I gave.

**Comment by mapx-** [comment]
if I got "success" means I clicked it

**Comment by aaronliu0130** [comment]
weird. lemme try a fresh install

**Comment by aaronliu0130** [comment]
Ok, I think the problem is notifications. You have to dismiss its request for notification permissions before clicking on the button.

**Comment by mapx-** [comment]
> notification 

you mean AG / fanboy's annoyances / cookie / popup lists ?

**Comment by aaronliu0130** [comment]
No. Like site permissions. It checks "remind me in 12 hours" by default and apparently it's sent through browser notifications.

**Comment by WildArticunoAppeared** [comment]
The same problem has now cropped up for me here. I tried a bunch of things like reinstalling uBlock, clearing the cache and cookies of the site, but it's still happening.

Months before, I had been getting the anti-adblock notification, but the Upvote Anyway button was working. The Upvote Anyway button has only stopped working last night.

**Comment by WildArticunoAppeared** [comment]
UPDATE/CLARIFICATION: I have been using Microsoft Edge. But I just installed Firefox, installed uBlock Origin as the only extension for that browser and tried again, only to be met with the same problem.

**Comment by LordTygon** [comment]
I have been having an issue with the site recently. You can click the upvote button and then it hangs for a moment and goes to a page asking to disable adblock. When you do, you are presented with an option to "upvote anyway," but when you click it, nothing happens. Disabling adblock allows clicking. I of course tried with all default settings. 
https://github.com/uBlockOrigin/uAssets/assets/4935642/5cd4b93d-6e63-4990-b99c-a0cece3206fb

This seems to be something recent. I do not recall this issue going back as far as November, so it may be a new issue or unrelated to OP's issue, but it appears something is borked currently.

**Comment by Daxy7** [comment]
@LordTygon Unable to reproduce. Take a capture of the logger by going into uBO's extension icon -> Icon beside the cog wheel -> Refresh the problematic page -> Maximize the logger window, and Toggle logger filtering, sort by 'blocked', 'allowed', 'modified' -> Take screenshot

**Comment by ItsProfessional** [comment]
I get detected even with uBO disabled - it's a site issue, contact the site.

**Comment by itslukej** [comment]
Hey - discordbotlist.com dev here

This is a recent bug where the "Upvote Anyway" button was unclickable - This has just been patched - Users can now freely use the site with adblocker without disabling it.  They'll just have to see the soft notice.

**Comment by mapx-** [comment]
@ffsmyusernameistaken

- no detection on my end
- reset uBO to defaults, don't add any other filter list, disable other extensions, test again

**Comment by WildArticunoAppeared** [comment]
It's happening again...

https://github.com/user-attachments/assets/1584d6cc-e5df-44ff-b2fa-d30ee3214d38
This time, I can Upvote Anyway on _some_ bots, but not all...

**Comment by ryanbr** [comment]
Just removed, since it was a new rule.

**Comment by WildArticunoAppeared** [comment]
And again....

https://github.com/user-attachments/assets/7e49379b-555a-4cca-8655-584681181de6

But this time, Upvote Anyway doesn't work.

**Comment by WildArticunoAppeared** [comment]
Do I have to make _another_ duplicate issue for this to get noticed?

**Comment by stephenhawk8054** [comment]
If you do that, we will ban you.

**Comment by uBlock-user** [comment]
> Do I have to make _another_ duplicate issue for this to get noticed?

Spamming will result in repo ban, beaware. Commenting is fine.

**Comment by WildArticunoAppeared** [comment]
I'm still getting this problem... even after updating the filter lists and deleting my cache... is anyone else still dealing with this?

**Comment by mapx-** [comment]
@WildArticunoAppeared 

test

```adb
@@||sb.scorecardresearch.com^$script,domain=discordbotlist.com
```

**Comment by WildArticunoAppeared** [comment]
Thanks! That works for the Upvote Anyway button! Though not all bots get the Upvote Anyway button...

https://github.com/user-attachments/assets/cec53819-3253-4d19-a9ef-e4a13becc91c

**Comment by mapx-** [comment]
provide examples without that Anyway button

**Comment by WildArticunoAppeared** [comment]
> provide examples without that Anyway button

Well, I just sent a screenshot of Pokétwo not getting the Upvote Anyway button in my previous post. Were you not able to see that image?
Dank Memer also has the same problem.

**Comment by stephenhawk8054** [comment]
Always. Give. Links.

Don't force us to type the URL or search manually from a screenshot.

**Comment by WildArticunoAppeared** [comment]
Sorry... didn't realise that...

https://discordbotlist.com/bots/poketwo/upvote
https://discordbotlist.com/bots/dank-memer/upvote

**Comment by mapx-** [comment]
I get no detection on upvoting (for the last links)

**Comment by ghost** [comment]
uBO gets detected if I want to upvote. I use Edge for Android.

https://github.com/user-attachments/assets/64c811b4-037f-4d80-b371-b69c03335c96

<details>

```yaml
uBlock Origin: 2025.906.66b0
Chromium Mobile: 139
filterset (summary):
 network: 163788
 cosmetic: 92847
 scriptlet: 34795
 html: 0
listset (total-discarded, last-updated):
 added:
  fanboy-cookiemonster: 46258-152, 18h.8m
  ublock-cookies-easylist: 5094-47, 18h.8m
 default:
  user-filters: 1-0, never
  ublock-filters: 41261-82, 2h.53m Δ
  ublock-badware: 11188-31, 2h.53m Δ
  ublock-privacy: 3470-4, 2h.53m Δ
  ublock-unbreak: 2680-1, 2h.53m Δ
  ublock-quick-fixes: 308-0, 2h.53m Δ
  adguard-mobile: 11340-67, 18h.10m
  easylist: 79454-290, 2h.53m Δ
  easyprivacy: 54471-662, 2h.53m Δ
  urlhaus-1: 32860-5, 2h.53m
  plowe-0: 3418-418, 18h.10m
  ROU-1: 1549-26, 17m
filterset (user): [array of 1 redacted]
userSettings:
 userFiltersTrusted: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1066 ms (selfie)
 maxAssetCacheWait: 56 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 13
 network:
  anonymised.io: 2
  btloader.com: 1
  cloudflareinsights.com: 1
  doubleclick.net: 1
  googlesyndication.com: 1
  googletagmanager.com: 2
  network-n.com: 1
  pbstck.com: 1
  permutive.app: 1
  prism.gg: 1
  scorecardresearch.com: 1
 extended:
  ##.ad
```

</details>

**Comment by ffsmyusernameistaken** [comment]
This was never fixed, I just gave up.

https://github.com/user-attachments/assets/e747bbd4-acce-470d-b592-b104e2369c8d
`https://discordbotlist.com/bots/poketwo/upvote`

**Comment by WildArticunoAppeared** [comment]
Hm, the detections stopped for me shortly after mapx's comment.

Maybe try updating your filters and clearing cache?

**Comment by ffsmyusernameistaken** [comment]
Nope, they always auto-update and nothing changed. Unless you use some additional filters that I don't?

**Comment by SuperDragon999** [comment]
I also have this issue on https://discordbotlist.com/bots/dank-memer/upvote

**Comment by SuperDragon999** [comment]
It only works after enabling only peter lowe's blocklist, but that blocklist doesn't block some ads.

**Comment by S8RAKI** [comment]
Having the same issue

**Comment by Alexielnoten** [comment]
To add to this, since it closed mine. SImilar issue. The adblocker is detected but rather than simply disabling the upvote button I get an error that says it can't open the gacha to prove your human, as well as stating you have to disable adblocker. I have no vpn, no other adblocker, etc. 

<img width="1788" height="1218" alt="Image" src="https://github.com/user-attachments/assets/75d9acfc-e0df-45c6-af56-e4f9adad75d0" />

Several on discord also report the same issue, so the issue is repeatable on several different devices. This is a recurring issue with discordbotlist.

**Comment by WildArticunoAppeared** [comment]
Yeah, I've also been getting the anti-adblocker notice, as well as the constant problems with the security step not finishing. No matter how many times I try, I'm not able to successfully upvote any bots.

**Comment by WildArticunoAppeared** [comment]
BTW one of the admins from the DBL Discord server has said that adblockers break the site and they have no intention of fixing it. So we might just be on our own this time.

**Comment by Alexielnoten** [comment]
> BTW one of the admins from the DBL Discord server has said that adblockers break the site and they have no intention of fixing it. So we might just be on our own this time.

I mean...part of ublock origin's mainstay is that they get around adblock sensors, which is why they are recommended for sites like youtube that will refuse to play videos unless you watch their 50 ads. So I never got the impression they ever fixed when adblockers broke their sites since they never really allowed them in the first place, hence the "no ad=no vote"
